Makefile

```diff
@@ -5,6 +5,9 @@ clean:
 lint:
 	flake8 tibanna
 
+build:
+	poetry install
```

Comment:
I'd add `make configure` here. You'll need to make a target.

And `install` looks redundant with `build`. Rather than eliminate it, to stay compatible, I suggest it should identify itself a synonym and then be implemented by `make build`.